### PR TITLE
Add an example  for creating a Zuul VM

### PR DESCRIPTION
**Pull request by majamassarini**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,3 +48,83 @@ Your locally cloned repo is: `project_dir: "{{ ansible_user_dir }}/{{ zuul.proje
 - Zuul/Gerrit questions & issues:
   - Internal IRC channel #rhos-ops
   - softwarefactory-operations-team@redhat.com
+
+## Zuul VM
+
+To debug issues related with the _Zuul VM_ you can follow this steps:
+
+- Install some _virt tools_
```
Comment (csomh) [inline README.md:56]:
```suggestion
- Install some _virt tools_ and Ansible:
```

File: README.md [hunk 2]
```diff
@@ -48,3 +48,83 @@ Your locally cloned repo is: `project_dir: "{{ ansible_user_dir }}/{{ zuul.proje
...
+
+```
+dnf install libvirt /usr/bin/virt-customize virt-manager
```
Comment (csomh) [inline README.md:59]:
```suggestion
$ sudo dnf install libvirt /usr/bin/virt-customize virt-manager ansible
```

File: README.md [hunk 3]
```diff
@@ -48,3 +48,83 @@ Your locally cloned repo is: `project_dir: "{{ ansible_user_dir }}/{{ zuul.proje
...
+-    src: /var/lib/nodepool/.ssh/zuul_rsa.pub
++    src: /home/maja/.ssh/id_rsa.pub
+     dest: "{{ image_tmp_dir }}/authorized_keys"
+     remote_src: true
+```
+
+- Build the _Zuul VM qcow2 image_; changing the `image_output` variable to a suitable location.
+
+```
+ansible-playbook -vvv -c local -i localhost, -e qcow2_type=true -e image_output=/home/maja/softwarefactory.io/zuul -bK nodepool/virt_images/cloud-fedora-35.yaml
```
Comment (csomh) [inline README.md:101]:
```suggestion
$ ansible-playbook -vvv -c local -i localhost, -e qcow2_type=true -e image_output=/home/maja/softwarefactory.io/zuul -bK nodepool/virt_images/cloud-fedora-35.yaml
```

File: README.md [hunk 4]
```diff
@@ -48,3 +48,83 @@ Your locally cloned repo is: `project_dir: "{{ ansible_user_dir }}/{{ zuul.proje
...
+  - File -> New Virtual Machine
+  - Choose Import existing disk image
+  - Browse -> Browse Local to the `image_output` dir and select _zuul.qcow2_
+  - Type _Fedora XX_ as your operating system type
+  - Create the new VM with the other wizard default settings
+
+- ssh into the VM like in the example; once the VM is started its **IP** is _displayed above the login prompt_
+
+```
+ ssh zuul-worker@192.168.122.253
```
Comment (csomh) [inline README.md:115]:
```suggestion
$ ssh zuul-worker@192.168.122.253
```

File: README.md [hunk 5]
```diff
@@ -48,3 +48,83 @@ Your locally cloned repo is: `project_dir: "{{ ansible_user_dir }}/{{ zuul.proje
...
+```
+
+- If you need to debug the hardly CI, as an example, copy the hardly project into the VM and enter the VM again:
+
+```
+scp -r /home/maja/PycharmProjects/hardly/ zuul-worker@192.168.122.253:/home/zuul-worker
+ssh zuul-worker@192.168.122.253
```
Comment (csomh) [inline README.md:122]:
```suggestion
$ scp -r /home/maja/PycharmProjects/hardly/ zuul-worker@192.168.122.253:/home/zuul-worker
$ ssh zuul-worker@192.168.122.253
```

File: README.md [hunk 6]
```diff
@@ -48,3 +48,83 @@ Your locally cloned repo is: `project_dir: "{{ ansible_user_dir }}/{{ zuul.proje
...
+```
+
+- Start the ansible playbook for the CI with something like the following
+
+```
+cd hardly
+ansible-playbook -vvv -c local -i localhost, -e '{"zuul": {"branch": "main", "project": {"src_dir": "/home/zuul-worker/hardly"}}}' files/zuul-tests.yaml
```
Comment (csomh) [inline README.md:129]:
```suggestion
$ cd hardly
$ ansible-playbook -vvv -c local -i localhost, -e '{"zuul": {"branch": "main", "project": {"src_dir": "/home/zuul-worker/hardly"}}}' files/zuul-tests.yaml
```

File: README.md [hunk 7]
```diff
@@ -48,3 +48,83 @@ Your locally cloned repo is: `project_dir: "{{ ansible_user_dir }}/{{ zuul.proje
 - Zuul/Gerrit questions & issues:
   - Internal IRC channel #rhos-ops
   - softwarefactory-operations-team@redhat.com
+
+## Zuul VM
```
Comment (csomh) [inline README.md:52]:
```suggestion
## Debug locally in a  Zuul VM
```

And maybe move it before the "More useful links" section.

**Review by jpopelka — APPROVED** [review]
So no try-wait-for-Zuul-error loop next time? :rocket: :1st_place_medal: :tada:

**Pull request merged**